standalone_apis_test.go

```diff
@@ -441,28 +520,37 @@ func TestGrantHandler(t *testing.T) {
 		nil,
 	)
 
-	t.Run("400 on missing resourceId from body", func(t *testing.T) {
+	t.Run("400 on missing body fields", func(t *testing.T) {
 		reqBody := setupGrantRequestBody(t, GrantRequestBody{
-			Subjects:    []string{"a"},
-			Groups:      []string{"b"},
-			Permissions: []string{"c"},
-			Roles:       []string{"d"},
+			ResourceID: "my-resource",
 		})
 		req, err := http.NewRequestWithContext(ctx, http.MethodPost, "/", bytes.NewBuffer(reqBody))
-		assert.NilError(t, err, "unexpcted error")
+		assert.NilError(t, err, "unexpected error")
 		w := httptest.NewRecorder()
 
 		grantHandler(w, req)
 
 		assert.Equal(t, w.Result().StatusCode, http.StatusBadRequest)
 	})
 
-	t.Run("400 on missing body fields", func(t *testing.T) {
+	t.Run("400 on missing resourceId from body if resourceType request param is present", func(t *testing.T) {
 		reqBody := setupGrantRequestBody(t, GrantRequestBody{
-			ResourceID: "my-resource",
+			Subjects:    []string{"a"},
+			Groups:      []string{"b"},
+			Permissions: []string{"c"},
+			Roles:       []string{"d"},
 		})
-		req, err := http.NewRequestWithContext(ctx, http.MethodPost, "/", bytes.NewBuffer(reqBody))
-		assert.NilError(t, err, "unexpcted error")
+
+		req := requestWithParams(
```

Comment:
follow the same indentation of other tests please

Reply:
indentation fixed